docs/invitations/index.md

```diff
@@ -1,10 +1,24 @@
 ---
 layout: "doc"
-title: "Sending invitations"
-abstract: "Triggering invite emails for new users"
+title: "User invitations"
+abstract: "Send magic invitations to new or existing users."
 ---
 
-### This document is in Draft
+## Use cases
+
+Magic invitations frictionlessly authenticate your users and allow them to make use of your application immediately.
+They can be sent to:
+
+- Add a colleague to review your documents
+- Share a photo with family members
+- Invite a new user to start using your service.
+
+## Steps
+
+1. Create a template for you invitation email.
```

Comment:
This is the step that is not the easiest to document. I think it's worth creating an invite email which say Alice invited Bob to service X